classes/02class/exercises/c02-network02/carloshz4/README.md

```diff
@@ -0,0 +1,25 @@
+# c02-network02
+
+## Commands Execution Output
+
+- Answers for `10.0.0.0/16`:
+
+|Subnet|Network|Host Range|Usable Hosts|Broadcast|AWS Reserved|
+|-|-|-|-|-|-|
+|subnet-a|10.0.0.0/18|10.0.0.1-10.0.63.254|16382|10.0.63.255|16379
+|subnet-b|10.0.64.0/18|10.0.64.1-10.0.127.254|16382|10.0.127.255|16379
+|subnet-c|10.0.128.0/18|10.0.128.1-10.0.191-254|16382|10.0.191.255|16379

```

Comment:
Small typo here on the IP `10.0.191-254`

Reply:
Thanks Daniel, I'll fix it.